src/applications/vaos/containers/AppointmentsPage.jsx

```diff
@@ -34,6 +38,13 @@ export class AppointmentsPage extends Component {
       scrollAndFocus();
     }
 
+    if (
+      this.props.showExpressCare &&
```

Comment:
Without EC, the fetching stays in here, otherwise it moves up to the AppointmentsPage component

Reply:
👍🏼 Assuming you mean the `FutureAppointmentsList`.

Reply:
Yes, I said that completely backwards